### ISSUE TITLE
[whatsapp] Can't see full sized contact profile image and group image

### ISSUE THREAD

**Issue by TronFortyTwo** [loqui/im#1194]
There is no way to open the normal resolution contact profile image and the group image. It will be an useful feature.

**Comment by nfsprodriver** [comment]
Good point, we should work on. May be in combination with the work on downloads...

**Comment by Honk2** [comment]
This is an UT feature, it is working on FFOS.

**Comment by nfsprodriver** [comment]
Yes, resolution and size very weird. Hard to find a sense on it.
